MapboxNavigation/NavigationViewController.swift

```diff
@@ -259,7 +285,7 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
         
         super.init(contentViewController: mapViewController, drawerViewController: tableViewController)
         
-        self.styles = styles ?? [DefaultStyle()]
+        self.styles = styles ?? [DefaultStyle(), DefaultDarkStyle()]
```

Comment:
`NavigationViewController(for:directions:styles:locationManager:)` <- styles here is optional to make it convenient to start a navigation without having to pass a style to the initializer. 

In Swift it allows you to use `NavigationViewController(for:directions:locationManager:)`, unfortunately, in Obj-C, you have to pass `styles:nil` but we need to apply a default style, otherwise it would just look like it looks in IB.